docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -14,7 +14,7 @@ The .NET SDK makes it possible to reduce the size of self-contained apps by [tri
 
 In .NET 6+, when publishing an app, the `PublishTrimmed` project file element produces trim analysis warnings for patterns that are not statically understood to be compatible with trimming, including patterns in your code and in dependencies.
 
-You will encounter detailed warnings originating from your own code and `ProjectReference` dependencies. You may also see warnings like `warning IL2104: Assembly 'SomeAssembly' produced trim warnings` for `PackageReference` libraries. This warning means that the library contained patterns that are not guaranteed to work in the context of the trimmed app, and may result in a broken app. Consider contacting the author to see if the library can be annotated for trimming.
+You will encounter detailed warnings originating from your own code and `ProjectReference` dependencies. You may also see warnings like `warning IL2104: Assembly 'SomeAssembly' produced trim warnings` for `PackageReference` libraries. This warning means that the library contained patterns that are not guaranteed to work in the context of the trimmed app, and may result in a broken app. Consider contacting the author to see if the library can be annotated for trimming. In order to view warnings throughout user and library code, set the `TrimmerSingleWarn` property to `false` and the full list of warnings will be displayed. The property can also be set from the command line, `/p:TrimmerSingleWarn=false`.
```

Comment:
```suggestion
You will encounter detailed warnings originating from your own code and `ProjectReference` dependencies. You may also see warnings like `warning IL2104: Assembly 'SomeAssembly' produced trim warnings` for `PackageReference` libraries. This warning means that the library contained patterns that are not guaranteed to work in the context of the trimmed app, and may result in a broken app. Consider contacting the author to see if the library can be annotated for trimming. In order to view warnings for your code and all of its dependencies, set the `TrimmerSingleWarn` property to `false` and the full list of warnings will be displayed. The property can also be set from the command line, `/p:TrimmerSingleWarn=false`.
```

I'm not sure "user and library code" makes sense here since this document is for library authors. So "library code" is a bit ambiguous.